cves/kernel/CVE-2022-3108.yml

```diff
@@ -167,8 +171,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: I don't see any evidence of automated discovery and it isnt really something where a user can directly interface with this function so crawling and finding it seems infeasible.
```

Comment:
I looked at the "CWE-252 : Unchecked return value" and it seems that automated static analysis could be used to detect this issue.

Reply:
I'll look into it.